sormas-app/app/src/main/res/layout/row_campaign_list_item_layout.xml

```diff
@@ -136,15 +138,15 @@
                 android:layout_marginRight="@dimen/listItemConentIndentRight"
                 >
 
-                <de.symeda.sormas.app.component.controls.ControlTextReadField
+                <TextView
                     android:id="@+id/campaignFormData_formDate"
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:textSize="@dimen/listActivityRowTertiaryTextSize"
                     android:textColor="@color/listActivityRowLeastText"
                     android:fontFamily="sans-serif"
-                    app:value="@{data.campaignFormMeta.creationDate}" />
+                    app:value="@{DateFormatHelper.formatLocalDate(data.campaignFormMeta.creationDate)}" />
```

Comment:
Please use data.formDate instead - currently this will actually display the date when the form definition was created (which is not really the information we need here)